packages/create-next-app/index.ts

```diff
@@ -99,69 +99,49 @@ async function run(): Promise<void> {
     process.exit(1)
   }
 
-  if (!program.example) {
-    const template = await prompts({
-      type: 'select',
-      name: 'value',
-      message: 'Pick a template',
-      choices: [
-        { title: 'Default starter app', value: 'default' },
-        { title: 'Example from the Next.js repo', value: 'example' },
-      ],
-    })
+  if (program.example === true) {
```

Comment:
if a user runs `yarn create next-app --example` then the `example` flag is true, otherwise it is the string of whatever they put after the flag.